src/plugins/intel_cpu/src/nodes/deconv.cpp

```diff
@@ -451,25 +456,61 @@ void Deconvolution::getSupportedDescriptors() {
     if (!fusedWith.empty()) {
         outputDataType = DnnlExtensionUtils::IEPrecisionToDataType(fusedWith[fusedWith.size() - 1]->getOriginalOutputPrecisionAtPort(0));
     }
-    if (getParentEdges().size() != (withBiases ? (biasPort + 1) : biasPort)) {
+    if (getParentEdges().size() != (deconvAttrs.withBiases ? (biasPort + 1) : biasPort)) {
         IE_THROW() << errorPrefix << " has incorrect number of input edges";
     }
     if (getChildEdges().empty()) {
         IE_THROW() << errorPrefix << " has incorrect number of output edges";
     }
+
+#if defined(OV_CPU_WITH_ACL)
+    NodeConfig config;
+    config.inConfs.resize(getParentEdges().size());
+    config.outConfs.resize(getOriginalOutputsNumber());
+
+    auto& creatorsMap = BlockedDescCreator::getCommonCreators();
+    for (size_t i = 0; i < getParentEdges().size(); ++i) {
+        auto checkDesc = [&](LayoutType format) -> bool {
+            NodeConfig config;
+            config.inConfs.resize(getParentEdges().size());
+            config.outConfs.resize(getOriginalOutputsNumber());
+
+            for (size_t i = 0; i < getParentEdges().size(); ++i) {
+                config.inConfs[i].setMemDesc(
+                        creatorsMap.at(format)->createSharedDesc(getOriginalInputPrecisionAtPort(i), getInputShapeAtPort(i)));
+            }
+            config.outConfs[0].setMemDesc(
+                    creatorsMap.at(format)->createSharedDesc(getOriginalOutputPrecisionAtPort(0), getOutputShapeAtPort(0)));
+
+            std::vector<MemoryDescPtr> srcMemoryDescs;
+            for (size_t i = 0; i < config.inConfs.size(); i++) {
+                srcMemoryDescs.push_back(config.inConfs[i].getMemDesc());
+            }
+            std::vector<MemoryDescPtr> dstMemoryDescs;
+            for (size_t i = 0; i < config.outConfs.size(); i++) {
+                dstMemoryDescs.push_back(config.outConfs[i].getMemDesc());
+            }
+
+            return AclDeconvExecutorBuilder::customIsSupported(deconvAttrs, srcMemoryDescs, dstMemoryDescs)
```

Comment:
Could you please clarify why do we need such sequence of the calls?
If it is necessary, please leave a comment with the detailed explanations